src/main/java/seedu/address/logic/commands/AssignCommand.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TUTORIAL;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.tutorial.Tutorial;
+
+import java.util.List;
+
+/**
+ * Assign a student to a tutorial by creating an tutorial object.
+ */
+public class AssignCommand extends Command {
+
+    public static final String COMMAND_WORD = "assign";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Assigns a student to a tutorial"
+            + "Parameters: "
+            + "INDEX (Must be a positive integer) "
+            + PREFIX_TUTORIAL + "TUTORIAL "
+            + "Example: " + COMMAND_WORD + " "
+            + "2"
+            + PREFIX_TUTORIAL + "Physics";
```

Comment:
Would it be better to show lowercase "physics" instead since the user might think that it is case-sensitive